includes/admin/class-wc-payments-admin.php

```diff
@@ -277,9 +306,19 @@ public function enqueue_payments_scripts() {
 		}
 
 		// TODO: Update conditions when ToS script is enqueued.
```

Comment:
The todo is probably not relevant anymore.

Reply:
Not after these checks: https://github.com/Automattic/woocommerce-payments/pull/993/commits/8ad28b977f3e8bb150397473aadadaf845edfb29#diff-3863b4f88d0515e080fadc56af3c5e38e942afb96e569cb000ae6a594e8bea4aR289